src/components/HomePage/HomePage.container.ts

```diff
@@ -1,9 +1,64 @@
 import { connect } from 'react-redux'
+import { TransactionStatus as TxStatus } from 'decentraland-dapps/dist/modules/transaction/types'
+import { TransactionStatus, TransactionType } from '../../modules/mana/types'
+import { getTransactionByNetwork } from '../../modules/mana/selectors'
 import { RootState } from '../../modules/reducer'
+
 import HomePage from './HomePage'
 import { MapStateProps, MapDispatchProps, MapDispatch } from './HomePage.types'
 
-const mapState = (_state: RootState): MapStateProps => ({})
+const mapStatus = (txStatus: TxStatus | null) => {
+  let resultStatus: TransactionStatus = TransactionStatus.PENDING
+  if (txStatus) {
+    if (txStatus === TxStatus.CONFIRMED) {
```

Comment:
Let's use these helpers from `decentraland-dapps` to know if the status is pending, failed or success (because there are more states like Queued and Replaced that are not being handled here):

```ts
import { isPending, hasFailed, hasSucceeded } from 'decentraland-dapps/dist/modules/transaction/utils.ts'
```